docs/examples/201/shared-image-gallery-with-image-defintition-and-role-assignment/main.bicep

```diff
@@ -0,0 +1,69 @@
+param azureSubscriptionID string
+param sigName string
+param sigLocation string
+param imagePublisher string
+param imageDefinitionName string
+param imageOffer string
+param imageSKU string
+param imageLocation string
+param roleNameGalleryImage string
+param principalID string
+param templateImageResourceGroup string
+
+var assignableScopes = '/subscriptions/${azureSubscriptionID}/resourcegroups/${templateImageResourceGroup}'

```

Comment:
since this is just a single value, I would say `assignableScope` or possibly something like `imageResourceGroupId` to be more specific.

I'm assuming that this resourceGroup could be in a different rg than the target of the deployment? Otherwise, you could use `resourceGroup().id`

Reply:
Thanks, that makes sense! I changed it to templateImageResourceGroupId to match the existing parameter templateImageResourceGroup.